src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -0,0 +1,267 @@
+import * as _ from "lodash";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Dictionary = _.Dictionary;
+import {TypeOfCancer, CancerStudy} from "../../api/CBioPortalAPI";
+import {ITreeDescriptor, default as DescriptorTreeNode} from "../tree/DescriptorTreeNode";
+import * as ReactBootstrap from 'react-bootstrap';
+import {BootstrapTable} from "react-bootstrap-table";

```

Comment:
I had already started using it when it was discussed and it is easy to swap out later. I spent minimal time working with the library.